Daramola98/WEconnectApp
# #159134214 Write tests for new features

#### What does this PR do?
This PR implement chore writing tests for new features

#### Description of Task to be completed?
- Write tests for user can edit and delete his/her review
- Complete testing of containers and e2e tests

#### How should this be manually tested?
- Setup project following instructions in readme then run the command npm run start:dev
- Run npm run test:client, all tests should be passing

#### What are the relevant pivotal tracker stories?
#159134214
